templates/piqa/templates.yaml

```diff
@@ -0,0 +1,121 @@
+dataset: piqa
+templates:
+  1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45: !Template
+    id: 1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45
+    jinja: 'Given a goal and a correct solution, generate a similar but wrong solution.
+
+      Goal: {{goal}}
+
+      Correct solution: {{[sol1, sol2][label]}}
+
+      Wrong solution:
+
+      |||
+
+      {{[sol1, sol2][1 - label]}}
+
+      '
+    name: Generate a similar but wrong solution
+    reference: ''
+  45773b41-4d13-4ade-9102-afe1543f5776: !Template
+    id: 45773b41-4d13-4ade-9102-afe1543f5776
+    jinja: 'Given a goal and 2 solutions, choose the most appropriate solution.
+
+      Goal: {{goal}}
+
+      {{"Solution 1"}}: {{sol2}}
+
+      {{"Solution 2"}}: {{sol1}}
+
+      Answer:
+
+      |||
+
+      {{[sol1, sol2][label]}}'
+    name: 'choose the most appropriate solution: reorder solution'
+    reference: ''
+  5f4b4645-9438-4375-9062-083130e6d04e: !Template
+    id: 5f4b4645-9438-4375-9062-083130e6d04e
+    jinja: "Given a goal and a wrong solution, rewrite it to give a correct solution.\n\
+      Goal: {{goal}} \nSolution: {{[sol1, sol2][1 - label]}}\nCorrected solution:\n\
+      |||\n{{[sol1, sol2][label]}}\n"
+    name: Correct the solution
+    reference: ''
+  99565244-4eaf-4004-a28b-4362ba5bcac3: !Template
+    id: 99565244-4eaf-4004-a28b-4362ba5bcac3
+    jinja: '{{goal}} {{sol2}}
+
+      Does this solution make sense?
+
+      |||
+
+      {{["No", "Yes"][label]}}'
+    name: Does this solution make sense? sol2
+    reference: ''
+  adfef248-f856-44fa-affd-e3223718854e: !Template
+    id: adfef248-f856-44fa-affd-e3223718854e
+    jinja: 'Given a goal and 2 solutions, choose the most appropriate solution.
+
+      Goal: {{goal}}
+
+      {{"Solution 1"}}: {{sol1}}
+
+      {{"Solution 2"}}: {{sol2}}
+
+      Answer:
+
+      |||
+
+      {{[sol1, sol2][label]}}'
+    name: choose the most appropriate solution
+    reference: ''
+  b5c69473-eedb-4c4f-a5fa-d4e266e43535: !Template
+    id: b5c69473-eedb-4c4f-a5fa-d4e266e43535
+    jinja: 'Given a sentence, correct it if it doesn''t make sense.
+
+      Input: {{goal}} {{sol2}}
+
+      Output:
+
+      |||
+
+      {{goal}} {{[sol1, sol2][label]}}
+
+      '
+    name: 'Correct the solution if false: from sol 2'
+    reference: ''
+  c8c45ef1-2ffc-43d7-8710-b98c2fc4f699: !Template
+    id: c8c45ef1-2ffc-43d7-8710-b98c2fc4f699
+    jinja: '{{goal}}
+
+      |||
+
+      {{[sol1, sol2][label]}}'
+    name: no prompt needed
+    reference: ''
+  f044def7-01c2-42de-b6ad-4e8c63ab2bf1: !Template
+    id: f044def7-01c2-42de-b6ad-4e8c63ab2bf1
+    jinja: '{{goal}} {{sol1}}
+
+      Does this solution make sense?
+
+      |||
+
+      {{["Yes", "No"][label]}}'
+    name: Does this solution make sense? sol1
+    reference: ''
+  f42cd457-a14b-465a-a139-d7d2407a3bac: !Template
+    id: f42cd457-a14b-465a-a139-d7d2407a3bac
+    jinja: 'Given a sentence, correct it if it doesn''t make sense.
+
+      Input: {{goal}} {{sol1}}
```

Comment:
This phrasing is a bit awkward with capitalization. Maybe you could make it more explicit? It looks like this:

```
Given a sentence, correct it if it doesn't make sense.
Input: When boiling butter, when it's ready, you can Pour it onto a plate
Output:```

or maybe just use {{sol1[0].lower() + sol[1:]}}

Reply:
Not sure if we should use `sol1[0].lower() + sol1[1:]}` everywhere, some solutions do need capitalization. But I guess lower is better. I updated most of the templates to use this.